src/components/posts/State.tsx

```diff
@@ -0,0 +1,15 @@
+import { atom, atomFamily } from 'recoil'
```

Comment:
There is no jsx rendered in this file, so it can be a `.ts` file instead of `.tsx`. Also, name of file could be `camelCase` or `dash-case` instead of `PascalCase` to further assist in telling these kind of files apart from component files.